service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/service/EncounterReportBundleService.java

```diff
@@ -106,7 +111,26 @@ private void addOrganization(Bundle bundle, Organization organization) {
         }
     }
 
-    private static void addEntry(Bundle bundle, Resource resource) {
+    private void addSubject(Bundle bundle, Encounter encounter) {
+        if (encounter.getSubjectTarget() instanceof Patient) {
+            Patient patient = (Patient) encounter.getSubjectTarget();
+            bundle.addEntry()
+                    .setFullUrl(patient.getIdElement().getValue())
+                    .setResource(patient);
+        }
+        if (encounter.getSubjectTarget() instanceof Group) {
+            Group group = (Group) encounter.getSubjectTarget();
+            bundle.addEntry()
+                    .setFullUrl(group.getIdElement().getValue())
+                    .setResource(group);
+            for (Group.GroupMemberComponent groupMemberComponent : group.getMember()) {
+                bundle.addEntry()
+                        .setFullUrl(groupMemberComponent.getIdElement().getValue())
+                        .setResource(groupMemberComponent.getEntityTarget());
+            }
+        }
+
+      private static void addEntry(Bundle bundle, Resource resource) {
```

Comment:
!!! IMPORTANT: there is a bracket missing above the method private static void addEntry() so it is not visible in the file as method. It would be compilation error. Have you run it locally? gradle test -i ?

Reply:
This error was caused by merge conflict.